.github/blog-template-demo.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Is this intended to be in markdown-ready format, or just collecting the information?

https://gohugo.io/content-management/front-matter/#front-matter-formats

We can add extra frontmatter definitions and extend the template to use them -- author byline + bio seems reasonable to me.

Reply:
hi @evankanderson this is intended to be a markdown-ready template. We can definitely add byline and bio as frontmatter definitions.

Reply:
I asked because I think the yaml fields in the frontmatter are case-sensitive, and are probably also work best if they are c-style identifiers (lowercase without spaces, starting with a letter).

Reply:
@evankanderson is there any AI here or can the comment be resolved?

Reply:
I think the AI is to lowercase all the YAML front-matter, and maybe change "Author handle" to "author_handle", though it looks like we're not using that anywhere.